jdaviz/configs/default/plugins/viewers.py

```diff
@@ -30,6 +32,51 @@ def __init__(self, *args, **kwargs):
         # Allow each viewer to cycle through colors for each new addition to the viewer:
         self.color_cycler = ColorCycler()
 
+    @property
+    def user_api(self):
+        # default exposed user APIs.  Can override this method in any particular viewer.
+        if isinstance(self, BqplotImageView):
```

Comment:
Hmm. What about Astrowidgets API?

Reply:
Should they be exposed here?

Reply:
I think this would be the place to expose them, but maybe not quite yet since you said they are still actively changing, and exposing them here would commit us to supporting/deprecating/etc?

Reply:
Hmm, I have to think about the implications here a bit more. It also won't do if we disallow people to use the API this way.

Reply:
> It also won't do if we disallow people to use the API this way.

Can you elaborate what you mean?

Reply:
I don't know yet. Lemme try to use it. Will let you know soon. 😆

Reply:
See https://github.com/spacetelescope/jdaviz/pull/2563#pullrequestreview-1751125032